resources/views/layouts/_head.blade.php

```diff
@@ -37,6 +37,7 @@
           integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 
     @if (file_exists(public_path('/css/app.css')))
-        <link href="{{ mix('/css/app.css') }}" rel="stylesheet">
+        <link rel="preload" href="{{ mix('/css/app.css') }}" as="style">
```

Comment:
Please add it after `<title>` tag - we want to tell the browser to load this file as soon as possible